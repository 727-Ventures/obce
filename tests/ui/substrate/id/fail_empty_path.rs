use obce::id;

fn main() {
    id!();
}
