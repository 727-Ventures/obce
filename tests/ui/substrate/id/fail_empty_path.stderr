error: unexpected end of input, expected identifier
 --> tests/ui/substrate/id/fail_empty_path.rs:4:5
  |
4 |     id!();
  |     ^^^^^
  |
  = note: this error originates in the macro `id` (in Nightly builds, run with -Z macro-backtrace for more info)
